### DISCUSSION TITLE
Associate Tags with Number of Usages

### DISCUSSION THREAD

**Discussion by da2ce7** [torrust/torrust-index-gui#282] [Feature requests]
Whenever a tag is displayed in the interface it should also display the number of torrents that are associated with it.

**Comment by da2ce7** [comment]
@josecelano proposed for tags especially in the administration screen.

**Reply by josecelano** [reply]
OK. I like to do the same as we did for categories. I think it's valuable info for the user since we do not have any statistics page.

![image](https://github.com/torrust/torrust-index-frontend/assets/58816/1eda21b8-96e4-4385-92b1-d706ca058d6b)

**Reply by josecelano** [reply]
Relates to: https://github.com/torrust/torrust-index-archive/issues/105
